tools/rum-models-generator/Sources/RUMModelsGeneratorCore/Input/Swift/SwiftType.swift

```diff
@@ -28,6 +28,11 @@ internal struct SwiftArray: SwiftType {
     var element: SwiftType
 }
 
+internal struct SwiftDictionary: SwiftType {
+    var key: SwiftType
+    var value: SwiftType
+}
```

Comment:
IMO we can save some control flows / tests if being more tailored on that, i.e.:
```swift
    var key: SwiftPrimitive<String>
    var value: SwiftPrimitiveType
```

Reply:
Yes you're right!